source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/Mappers/MeteringPointTypeMapper.cs

```diff
@@ -20,7 +20,16 @@ namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.SqlState
 
 public static class MeteringPointTypeMapper
 {
-    public static MeteringPointType FromDeltaTableValue(string timeSeriesType) =>
+    public static MeteringPointType? FromDeltaTableValue(string meteringPointType) =>
+        meteringPointType switch
+        {
+            "production" => MeteringPointType.Production,
+            "consumption" => MeteringPointType.Consumption,
+            "" => null,
+            _ => throw new NotImplementedException($"Cannot map meteringPointType type '{meteringPointType}"),
```

Comment:
Notice: If we want the following change, we might wanna make the same "fix" for "timeSeriesType" further below.

I expect we want to surround the type with ' like the following:
```suggestion
            _ => throw new NotImplementedException($"Cannot map meteringPointType type '{meteringPointType}'"),
```